apps/web/components/Proposals.tsx

```diff
@@ -251,28 +249,53 @@ export function Proposals({
     return encodedProposalId;
   };
 
-  //test executing a proposal with distribute function
+  //executing proposal distribute function / alert error if not executable / notification if success
   const {
     data: distributeData,
     write: writeDistribute,
     error: errorDistribute,
     isSuccess: isSuccessDistribute,
+    isError: isErrorDistribute,
```

Comment:
i am a bit confused with the naming here, which is the difference between error: errorDistribute and then we have another isError: isErrorDistribute? just by the naming seems that is the same maybe we can find a better naming for those

Reply:
The naming it's just following the pattern, "is" in the beginning indicate that it's boolean, and the other is the error it self. Some checking types get it more clear.